README.md

```diff
@@ -40,6 +40,51 @@ POSTGIS_VERSIONS="" make build
 
 For further environment variables that can be set, we point you to the [Makefile](Makefile) itself.
 
+# Verify your work
+
+For every pushed commit to this repository, a Docker Image will be built and is available at a private
+Amazon Elastic Container Registry (ECR).
+
+## Find the image tag for your commit
+
+> Replace *** with the AWS Account ID.
+
+Once your commit is pushed, a Docker Image will be built, and if successful, will be pushed.
+The tag of this Docker Image will be `cicd-<first 8 chars of commit sha>`, for example, for commit `baddcafe...`, the tag will look like:
+
+    ***.dkr.ecr.us-east-1.amazonaws.com/timescaledb-ha:cicd-deadbeef
```

Comment:
```suggestion
    ***.dkr.ecr.us-east-1.amazonaws.com/timescaledb-ha:cicd-baddcafe
```

Reply:
What are those wildcards in `***.dkr.ecr.us-east-1.amazonaws.com`, is this a template to fill in with the actual hostname?

Reply:
It's our private AWS account id. Although not secret, it is advised to keep it private, background:

https://security.stackexchange.com/questions/192539/keeping-aws-account-id-secret

Reply:
Perhaps being a bit too cautious though, however, once a value is leaked, it cannot be unleaked ...

Reply:
I agree, thank you for making it clear.